cheroot/connections.py

```diff
@@ -162,7 +162,7 @@ def _expire(self):
         """
         # find any connections still registered with the selector
         # that have not been active recently enough.
-        threshold = time.time() - self.server.timeout
+        threshold = now - self.server.timeout
```

Comment:
It reads weird to see `now` passed as an arg. I think it may be easier to follow if you pass the `threshold` as an arg instead.